src/components/layout/BottomNavigation.tsx

```
@@ -0,0 +1,330 @@
+import { toast } from "sonner";
+import { isAddress } from "viem";
+import { ConnectButton } from "@rainbow-me/rainbowkit";
+
+interface BottomNavigationProps {
+  isConnected: boolean;
+}
+
+const BottomNavigation: React.FC<BottomNavigationProps> = ({ isConnected }) => {
+  const pathname = usePathname();
+  const router = useRouter();
+  const {} = useAccount();
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)